Detectors/MUON/MCH/Conditions/src/DCSAliases.cxx

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MCHConditions/DCSAliases.h"
+
+#include "DetectionElements.h"
+#include "HVAliases.h"
+#include "LVAliases.h"
+#include <array>
+#include <fmt/core.h>
+#include <iostream>
+#include <map>
+#include <optional>
+#include <set>
+
+namespace
+{
+
+bool in(o2::mch::dcs::MeasurementType type,
+        const std::vector<o2::mch::dcs::MeasurementType>& types)
+{
+  return std::find(begin(types), end(types), type) != types.end();
+}
+
+} // namespace
+
+namespace o2::mch::dcs
+{
+
+std::vector<std::string> aliases(std::vector<MeasurementType> types)
+{
+  std::vector<std::string> aliasesVector;
+
+  if (in(MeasurementType::HV_V, types)) {
+    aliasesVector.insert(begin(aliasesVector),
```

Comment:
This is likely a detail, but is there a particular reason to insert at the front of the vector? It might be faster to insert at the back.

Reply:
no special reason indeed. will change.